.circleci/config.yml

```
@@ -666,7 +666,7 @@ jobs:
             - frontend/node_modules
       - run:
           working_directory: *workspace_frontend
-          command: yarn lint
+          command: yarn lint-strict
```

You can actually just do:
```suggestion
          command: yarn lint --max-warnings 0
```

as ESLint parses options even after the target files list but either way is fine with me.